app/controllers/person_relations_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class PersonRelationsController < CrudController
+  def index
+    redirect_to person_path(entry.person)
+  end
+
+  def show
+    redirect_to person_path(entry.person)
+  end
+
+  def create
+    super(:location => person_path(person)) do |format, success|
+      format.turbo_stream { render 'save_and_new' } if success && params.key?(:'save-and-new')
+    end
+  end
+
+  def update
+    super(:location => person_path(person))
+  end
+
+  def destroy
+    super(:location => person_path(person)) do |format, success|
+      format.turbo_stream { render turbo_stream: turbo_stream.remove(entry) } if success
+    end
+  end
+
+  private
+
+  def entry
+    relation = super
+    relation.person = person unless person == relation.person
```

Comment:
shouldn't it just raise an exception if person.id is relation.person_id?

shouldn't this be correctly assigned by assign_attributes (dry crud)